plugins/fileservices/fileservices.cpp

```diff
@@ -1031,6 +1055,9 @@ FILESERVICES_API char * FILESERVICES_CALL implementSprayXml(ICodeContext *ctx, c
     if (noSplit)
         req->setNosplit(true);
 
+    if (keepSourceEncoding)
+        req->setKeepSourceEncoding(true);
```

Comment:
I think you need an else statement here (as with code in implementSprayVariable) to call setKeepSourceEncoding(false), because otherwise servers will assume keep source formatting.

Reply:
Else added.